tools/pdp-metadata/pdp-metadata.js

```diff
@@ -239,14 +240,15 @@ async function addVariantsToProducts(products, config) {
   products.forEach(({ productView: metaData, variants }) => {
     data.push(
       [
-        metaData.path, // URL
+        metaData.path.toLowerCase(), // URL
         metaData.meta_title, // title
         metaData.meta_description, // description
         metaData.meta_keyword, // keywords
+        metaData.sku, //sku
```

Comment:
This change is to align the `pdp-metadata` generator tool to the output of the prerender overlay as the overlay adds `sku` to the _lower case_ product path.

Outcome of this change is that products will only be on lowercase urls, so we may need to change wherever we are linking to the pages as well.